### ISSUE TITLE
Change of query URL

### ISSUE THREAD

**Issue by kfinlay** [fractaledmind/alfred_libgen#1]
I've been having problems accessing libgen.org URLs using the workflow. http://gen.lib.rus.ec recommends using libgen.in . I changed this in the workflow, and now the pdf URLs work fine. If you're having the same problems, could you change this here and on packal?

This workflow is brilliant. I use it even for books I own because it makes citations and quick lookups super easy.

Thanks, Keith

**Comment by fractaledmind** [comment]
Good point. Make a simple Pull Request and I'll update Packal with the new code.

**Comment by kfinlay** [comment]
I'm going to hold off. I tried the libgen.in URL and now it is blocked by my ISP. It seems like whack-a-mole. An alternative framework might just point to the http://gen.lib.rus.ec results page and give the user a download option there, since that URL seems more robust. In any case, I'd hate to fiddle with the script if it's just my ISP causing problems.
